unittests/snapshot_tests.cpp

```diff
@@ -382,10 +382,10 @@ void compatible_versions_test()
    std::filesystem::copy(source_log_dir / "blocks.index", config.blocks_dir / "blocks.index");
    TESTER base_chain(config, *genesis);
 
-   std::string current_version = "v7";
+   std::string current_version = "v8";
 
    int ordinal = 0;
-   for(std::string version : {"v2", "v3", "v4" , "v5", "v6", "v7"})
+   for(std::string version : {"v2", "v3", "v4", "v5", "v6", "v8"}) // v7 version not supported in spring 1.0.1 and above
```

Comment:
Capitalize Spring.

Reply:
Also, maybe change the example in the comment below of how to support a new snapshot version in this test to use `snap_v8` instead of `snap_v7`.

Reply:
https://github.com/AntelopeIO/spring/pull/719/commits/edf294dee52b5f314bab2a52928de9d2a06a1eb6